### DISCUSSION TITLE
Enabling Independent Real-Time Chat Between Clients

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [socketio/socket.io#5254] [Q&A]
Hi Community, Is it possible to do real-time chat between any two clients independently using socket.io, similar to a real-time chat application?

**Comment by darrachequesne** [comment]
Hi! Yes of course.

Please check [our Chat platform example](https://github.com/socketio/socket.io-chat-platform/), which includes private messaging.

**Reply by amoljagdalepucsd** [reply]
Thank you for the response @darrachequesne ; it's a great platform. Is screen sharing (browser/entire screen) supported in Socket.io? I’ve been checking the code but am not sure how to integrate it using WebRTC. It would be great if you could help me with this or provide some suggestions to keep me on track.
